src/shared/logs/logs.py

```diff
@@ -191,6 +200,27 @@ def get_raw_events(
                 )
             )
 
+        maintainer_qs = self._annotate_username(
+            MaintainersEditEvent.objects.prefetch_related(
+                "pgh_context", "maintainer"
+            ).filter(suggestion__in=suggestion_ids)
+        )
+
+        for maintainer_event in maintainer_qs.all().iterator():
+            raw_events.append(
+                MaintainerChangeEvent(
+                    suggestion_id=maintainer_event.suggestion.id,
+                    timestamp=maintainer_event.pgh_created_at,
+                    username=maintainer_event.username,
+                    action=maintainer_event.pgh_label,
+                    # TODO: we should use Pydantic model for maintainers and
+                    # friends as well, at some point
+                    maintainer=cast(
+                        Maintainer, model_to_dict(maintainer_event.maintainer)
```

Comment:
<img width="849" height="307" alt="image" src="https://github.com/user-attachments/assets/6103c3c3-2747-4a87-9a38-a2ca3d0135f3" />

`model_to_dict` will trip up on an unset `email` in the [`Maintainer` model](https://github.com/Nix-Security-WG/nix-security-tracker/pull/562/files#diff-120d76390978d1d6de1ddd431d367056180c1c0e7cb522e53e91bab736fe786bR58)

Reply:
Added a quick fix